### DISCUSSION TITLE
Empty admin/files directory when trying to use Memories App and Preview Generator

### DISCUSSION THREAD

**Discussion by aleyoscar** [nextcloud/all-in-one#2524] [Questions] [answered]
All of my files are located on external storages, and when trying to use the [Memories](https://memories.gallery/) app I need to run an occ command which scans my directory for images and indexes them. When I run `occ memories:index` though I get "Skipping folder /admin/files due to .nomedia file". The same also happens when I try to index my images using [Preview Generator](https://github.com/nextcloud/previewgenerator). When I run `occ preview:generate-all` I get "Skipping folder /admin/files".

When I look in the admin/files folder using `sudo ls -la /var/lib/docker/volumes/nextcloud_aio_nextcloud_data/_data/admin/files` it is indeed empty, and no .nomedia or .noimage are present.

I'm assuming this is because I have everything in external storages? Which is why my admin/files directory is empty? But how can I make these apps scan my external storages? The apps seem to suggest that it is possible to use external storages for photos but I can't get it to work. Any help would be appreciated.

Thanks!

**Comment by szaimen** [comment, marked as answer]
Hi, apparently there is a .nomeida file in your external storage that you need to remove in order to make it work.

**Reply by aleyoscar** [reply]
Yes, I had tried removing the .nomedia file from the server using the command line, but (obviously) Nextcloud re-synced it and loaded it back. After removing it using the web app the sync worked. Thanks!
